jobs/JGFS_ATMOS_POSTSND

```diff
@@ -21,17 +21,20 @@ export DBNROOT=${DBNROOT:-${UTILROOT}/fakedbn}
 ##############################
 export DBNROOT=${DBNROOT:-${UTILROOT}/fakedbn}
 
-YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_ATMOS_HISTORY COM_ATMOS_BUFR \
-  COM_ATMOS_WMO COM_ATMOS_GEMPAK
-
-if [[ ! -d "${COM_ATMOS_BUFR}" ]]; then
-    mkdir -p "${COM_ATMOS_BUFR}"
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
+  COMIN_ATMOS_HISTORY:COM_ATMOS_HISTORY_TMPL \
+  COMOUT_ATMOS_BUFR:COM_ATMOS_BUFR_TMPL \
+  COMOUT_ATMOS_WMO:COM_ATMOS_WMO_TMPL \
+  COMOUT_ATMOS_GEMPAK:COM_ATMOS_GEMPAK_TMPL
```

Comment:
What are these directories COM_ATMOS*_TMPL for? The names are not consistent, for example, COMIN_ATMOS_HISTORY vs COM_ATMOS_HISTORY_TMPL? @KateFriedman-NOAA

Reply:
@BoCui-NOAA 
`declare_from_tmpl` creates the variable `COMIN_ATMOS_HISTORY` based on the variable template defined under `COM_ATMOS_HISTORY_TMPL`.  `COM_ATMOS_HISTORY_TMPL` is declared in `parm/config/gfs/config.com`.  
The result of this call in effect is:
```bash
export COMIN_ATMOS_HISTORY=$(compath.py ${envir}/com/gfs/${gfs_ver})"/${RUN}.${PDY}/${cyc}/model/atmos/history"
```

Reply:
Git it now, Thanks!